File: MyApplication3/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.AutoCompleteTextView;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    String[] rating = {"5","4","3","2","1"};


    AutoCompleteTextView autoCompleteTextView;

    ArrayAdapter<String> adapterItems;



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        autoCompleteTextView = findViewById(R.id.menu_text);
        adapterItems = new ArrayAdapter<String>(this, R.layout.list_item,rating);

        autoCompleteTextView.setAdapter(adapterItems);

        autoCompleteTextView.setOnItemClickListener(new AdapterView.OnItemClickListener(){
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int i, long l) {
                String item = parent.getItemAtPosition(i).toString();
                Toast.makeText(MainActivity.this,"Rating: " + item,Toast.LENGTH_SHORT).show();
            }
        });

        Button buttonSubmit = findViewById(R.id.button);
        buttonSubmit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // Code to execute when the button is clicked
                Toast.makeText(MainActivity.this, "Submitted Successfully", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: app/src/main/java/com/codepath/zipfix/MainActivity.kt
package com.codepath.zipfix

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.parse.ParseObject
import android.widget.Button
import com.parse.ParseUser


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val current_user = ParseUser.getCurrentUser()
        val login_log = ParseObject("logsLogins")
        login_log.put("user", current_user.getUsername())
        login_log.saveInBackground()

        findViewById<Button>(R.id.logout_button).setOnClickListener {
            logout_user()
        }
    }
    private fun logout_user() {
        ParseUser.logOut()
        return_to_login()
    }

    private fun return_to_login() {
        val intent = Intent(this@MainActivity, LoginActivity::class.java)
        startActivity(intent)
    }
}
